Lib9c/Action/ClaimItems.cs

```diff
@@ -0,0 +1,134 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Linq;
+using Bencodex.Types;
+using Libplanet.Action;
+using Libplanet.Action.State;
+using Libplanet.Crypto;
+using Libplanet.Types.Assets;
+using Nekoyume.Action;
+using Nekoyume.Extensions;
+using Nekoyume.Model.Item;
+using Nekoyume.Model.State;
+using Nekoyume.TableData;
+using static Lib9c.SerializeKeys;
+
+namespace Nekoyume.Action
+{
+    public class ClaimItems : GameAction, IClaimItems
+    {
+        private const string ActionTypeText = "claim_items";
+
+        public IEnumerable<Address> AvatarAddresses { get; private set; }
+        public IEnumerable<FungibleAssetValue> Amounts { get; private set; }
```

Comment:
I think it is not good to use  `IEnumerable<T>` as properties' type. Of course, it will be good for each case. 

`IEnumerable<T>` doesn't guarantee it returns the same value for every iteration.

https://dotnetfiddle.net/G3jdWx

```csharp
using System;
using System.Globalization;
using System.Collections.Generic;
using System.Linq;
					
public class Program
{
	public static void Main()
	{
		int i = 1;
		IEnumerable<string> enumerable = new string[] { "foo", "bar" }.Select(x => string.Concat(Enumerable.Repeat(x, i++)));
		foreach (var s in enumerable) {
			Console.WriteLine(s);
		}

		foreach (var s in enumerable) {
			Console.WriteLine(s);
		}
	}
}
```

The above example code's output...:

```
foo
barbar
foofoofoo
barbarbarbar
```